lib/modals/add_points_form.dart

```diff
@@ -72,33 +77,116 @@ class _AddPointsFormState extends ConsumerState<AddPointsForm> {
             },
             controller: pointsController,
           ),
+          const SizedBox(height: 10),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              SuggestedPointsButton(
+                  pointsController: pointsController,
+                  points: "-10",
+                  color: Colors.red),
+              SuggestedPointsButton(
+                  pointsController: pointsController,
+                  points: "-5",
+                  color: Colors.red),
+              SuggestedPointsButton(
+                  pointsController: pointsController,
+                  points: "5",
+                  color: Colors.green),
+              SuggestedPointsButton(
+                  pointsController: pointsController,
+                  points: "10",
+                  color: Colors.green),
+            ],
+          ),
+          const SizedBox(height: 10),
           ElevatedButton(
             onPressed: () async {
               if (_formKey.currentState!.validate()) {
-                final response = await ref.watch(firebaseWriterServiceProvider).setPointsToTeam(
-                      widget.eventId,
-                      selectedTeam,
-                      widget.round,
-                      int.parse(pointsController.value.text),
-                    );
-
-                String responseText = response.fold(
-                  (error) => "Failed to set points",
-                  (success) =>
-                      "Successfully set ${pointsController.value.text} points to $selectedTeam",
-                );
+                Either<String, String> response = await updateTeamPoints();
```

Comment:
can this be just string?

Reply:
I'm a big fan of the Either class 
https://medium.com/nerd-for-tech/better-error-handling-with-either-type-in-dart-b91bef20d716